test/jdk/java/lang/invoke/TEST.properties

```diff
@@ -1,2 +1,5 @@
 # disabled till JDK-8219408 is fixed
 allowSmartActionArgs=false
+
+# Reflection methods for primitive classes are in PrimitiveClass instead of java.lang.class
+modules = java.base/jdk.internal.value:+open
```

Comment:
which test needs to reflective access to `jdk.internal.value.PrimitiveClass`?   The methods are public and I don't expect they need any reflective access.